Create New-CsTeamsMessagingPolicy/New-CsTeamsMessagingPolicy.ps1

```diff
@@ -0,0 +1,18 @@
+$logfile = "C:\CreateNew-CsTeamsMessagingPolicylog_$(get-date -format `"yyyyMMdd_hhmmsstt`").txt"
```

Comment:
Please add Synopsis block, i.e. from this template for inline help. https://github.com/TeamsAdminSamples/PowerShell/blob/main/z_Templates/New-PowerShell.ps1